[TASK] Add hint to LinkHandler API regarding constants (#4052)
TYPO3-Documentation/TYPO3CMS-Reference-CoreApi: Documentation/ApiOverview/LinkHandling/Linkhandler/Index.rst

in class :php:`\TYPO3\CMS\Backend\LinkHandler\RecordLinkHandler`
which does not contain Hooks.

.. attention::

    It is important, that the `storagePid` is hard coded in TSConfig, because using

>     constants, e.g. from the site configuration, won't work here.

Reviewer: ```suggestion
    constants, for example from the site configuration, will not work here.
```
